stories/filterinput.stories.js

```diff
@@ -0,0 +1,10 @@
+import FilterInput from "../src/components/FilterInput.svelte";
+
+export default {
+  title: "FilterInput",
+};
+
+export const Basic = () => ({
+  Component: FilterInput,
+  props: "",
+});
```

Comment:
Can you add some examples of a simple list of items being filtered? e.g.

"foo", "bar", "foobar"

Reply:
Alright. Adding in the next commit.